justfile

```diff
@@ -150,7 +150,7 @@ build *args:
 
 # List all services and their URLs and ports
 ps:
-    python3 utilities/ps.py
+    python3 utilities/ps.py || true
```

Comment:
A comment on why we're silently exiting with code 0 would be nice. I don't know if there are any recipes that depend on `ps`.

Reply:
Great call, I'll add that!